.github/workflows/cron.yml

```diff
@@ -0,0 +1,30 @@
+name: cron
+
+on:
+  schedule:
+  # Run every day at 00:00 UTC
+  - cron: 0 0 * * *
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
+        env:
+        - TOXENV: pipmaster
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          version: ${{ matrix.python-version }}
+      - name: Install dependencies
```

Comment:
Are you really? You are installing tox and then tox installs the dependencies.
I would just drop the name. GH will display the command, is descriptive enough I guess.

Reply:
Blindly copied from the default template. Maybe "Install tox" is good enough?

Reply:
Sure

Reply:
Addressed in 9962ce3.